### PR TITLE
Add Metasploit nightly

### PR DESCRIPTION
**Pull request by nbsantos**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/metasploit.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'metasploit' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/metasploit.rb]: Give it a version and a shasum. Since it has an appcast, it can have those.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/metasploit.rb (hunk 1 above)
Comment (nbsantos) [inline Casks/metasploit.rb]: Added the version and shasum as requested. I also updated the app cast checkpoint to reflect the latest one as of today.

File: Casks/metasploit.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  url 'https://osx.metasploit.com/metasploitframework-latest.pkg'
+  appcast 'https://github.com/rapid7/metasploit-framework/releases.atom',
+          checkpoint: '5de0fa55d535d005b078370eaaf3e50b4428a25041fdae604e4e0d66413595a6'
+  name 'Metasploit'
+  homepage 'https://www.metasploit.com'
+  license :bsd
+
+  pkg 'metasploitframework-latest.pkg'
+
+  uninstall script:  { executable: 'uninstall.tool', input: %w[Yes] },
```
Comment (vitorgalvao) [inline Casks/metasploit.rb]: Split this into lines (see other casks for examples, I’m currently on a phone).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Closing for lack of reply. If you still wish to include this, please resubmit with the requested fixes.

**Pull request closed without merging**